pkg/backend/snapshot.go

```diff
@@ -672,7 +675,7 @@ func (sm *SnapshotManager) saveSnapshot() error {
 // defaultServiceLoop saves a Snapshot whenever a mutation occurs
 func (sm *SnapshotManager) defaultServiceLoop(mutationRequests chan mutationRequest, done chan error) {
 	// True if we have elided writes since the last actual write.
-	hasElidedWrites := false
+	hasElidedWrites := true
```

Comment:
Not really familiar with the code, but why do we default to true here now?

Reply:
Because it turns out that if you don't the tests can fail because the snapshot never wrote anything if it thought everything was an elided write!